pkgs/development/python-modules/homf/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  # pytestCheckHook,
+  pythonOlder,
+
+  hatchling,
+  packaging,
+}:
+
+buildPythonPackage rec {
+  pname = "homf";
+  version = "1.0.0";
+  pyproject = true;
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "duckinator";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-PU5VjBIVSMupTBh/qvVuZSFWpBbJOylCR02lONn9/qw=";
+  };
+
+  build-system = [ hatchling ];
+
+  pythonRelaxDeps = [ "packaging" ];
+
+  dependencies = [ packaging ];
+
+  pythonImportsCheck = [
+    "homf"
+    "homf.api"
+    "homf.api.github"
+    "homf.api.pypi"
+  ];
+
+  # There are currently no checks which do not require network access, which breaks the check hook somehow?
+  # nativeCheckInputs = [ pytestCheckHook ];
+  # pytestFlagsArray = [ "-m 'not network'" ];
+
+  meta = with lib; {
+    description = "Asset download tool for GitHub Releases, PyPi, etc.";
+    mainProgram = "homf";
+    homepage = "https://github.com/duckinator/homf";
+    maintainers = with maintainers; [ nicoo ];
+    platforms = platforms.all;
```

Comment:
```suggestion
```
Since it does not work on platforms not supported by nixpkgs python, it should adopt the python default.